### PR TITLE
Test merge

### PR DESCRIPTION
**Pull request by ehuss**

### CODE REVIEW

**Comment by ehuss** [comment]
Unknown labels: S-waiting-on-reviewz

**Comment by ehuss** [comment]
This repository does not allow merge commits.
Your PR cannot be merged until it is rebased.
The following commits are merge commits:
- 8f03c06a4c340f93589bfe14a7ac1d64f519dfe9
- 82dae1d30844e82f4be89e8ec313ac7931520d49

**Comment by ehuss** [comment]
This repository does not allow merge commits.
Your PR cannot be merged until it is rebased.
The following commits are merge commits (since this message was last posted):
- 3d3382fdb4c222b1a3b0c6c5f8239c226f680ecb

**Comment by ehuss** [comment]
This repository does not allow merge commits.
Your PR cannot be merged until it is rebased.
The following commits are merge commits (since this message was last posted):
- 48a1d4e8879ca3c81c5e1e6879cfb7f00f7a0b90

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # triagebot-test
 Triagebot testing
+test
```
Comment (grashgal) [inline README.md:3]: Test

Comment (grashgal) [inline README.md:3]: test

Comment (grashgal) [inline README.md:3]: test

**Comment by ehuss** [comment]
:umbrella: The latest upstream changes made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

**Pull request closed without merging**